neo/SmartContract/ContractManifestFeatures.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Neo.SmartContract
+{
+    /// <summary>
+    /// The features field describes what features are available for the contract.
+    /// </summary>
+    public class ContractManifestFeatures
```

Comment:
https://github.com/neo-project/neo/blob/master/neo/Ledger/ContractPropertyState.cs

We can reuse `ContractPropertyState` for now.